CI pipeline doesn't seem to be running pytest right now

There is currently a failing unit test in `tests/test_basics.py`.

It seems that the pytest unit tests aren't being run in this repo's CI workflow right now. It would be great if it did! 

```
________________________________________________________ test_scriptsession ________________________________________________________

    def test_scriptsession():
        session = ScriptSession()
        session.eval('integer arr[string] = \'{"Hello":4, "World":8, default:-1};')
        session.eval(
            """
    function int func(int i, integer arr[string]);
        case (i) inside
            [32'sd1:32'sd2]: return 1;
            arr: return 2;
        endcase
        return 3;
    endfunction
    """
        )
    
        assert session.eval("func(4, arr)").value == 2
>       assert len(session.getDiagnostics()) == 0
E       assert 1 == 0
E        +  where 1 = len(<pyslang.Diagnostics object at 0x71edac84f5f0>)
E        +    where <pyslang.Diagnostics object at 0x71edac84f5f0> = getDiagnostics()
E        +      where getDiagnostics = <pyslang.ScriptSession object at 0x71edac84e870>.getDiagnostics

tests/test_basics.py:102: AssertionError
===================================================== short test summary info ======================================================
FAILED tests/test_basics.py::test_scriptsession - assert 1 == 0
```

--- a/tests/test_basics.py
+++ b/tests/test_basics.py
@@ -85,7 +85,7 @@ source:3:23: warning: arithmetic between operands of different types ('logic' an
 
 def test_scriptsession():
     session = ScriptSession()
-    session.eval('integer arr[string] = \'{"Hello":4, "World":8, default:-1};')
+    session.eval("""integer arr[string] = '{"Hello":4, "World":8, default:-1};""")
     session.eval(
         """
 function int func(int i, integer arr[string]);
@@ -99,7 +99,10 @@ endfunction
     )
 
     assert session.eval("func(4, arr)").value == 2
-    assert len(session.getDiagnostics()) == 0
+
+    # Missing 'default' in case statement. Check that the diagnostic is reported.
+    assert len(session.getDiagnostics()) == 1
+    assert session.getDiagnostics()[0].code == Diags.CaseDefault
 
 
 def test_symbol_inspection():

